mirrord/cli/src/error.rs

```diff
@@ -327,6 +329,10 @@ pub(crate) enum CliError {
     #[error("mirrord returned a target resource of unknown type: {0}")]
     #[diagnostic(help("{GENERAL_BUG}"))]
     OperatorReturnedUnknownTargetType(String),
+
+    #[error("An error occurred in the port-forwarding process: {0}")]
+    #[diagnostic(help("{GENERAL_BUG}"))]
+    PortForwardingError(PortForwardError),
```

Comment:
```suggestion
    PortForwardingError(#[from] PortForwardError),
```
Instead of manually implementing `From<PortForwardError> for CliError` ^^